app/helpers/comments_helper.rb

```diff
@@ -286,6 +286,14 @@ def tag_comment_as_spam_link(comment)
     end
   end
 
+  # gets the css user-<id> class name for the comment
+  def comment_user_css_class(comment)
+    return "" if comment.pseud.nil?
+    return "" if comment.pseud.user.is_author_of?(comment.ultimate_parent) && comment.ultimate_parent.anonymous?
+
+    " user-#{comment.pseud.user.id}"
```

Comment:
Let's leave out the leading space here for consistency with [css_classes_for_creation_blurb](https://github.com/otwcode/otwarchive/blob/84f2f1eed7d5e254411116417c1500b4a31ae7e7/app/helpers/application_helper.rb#L601-L609). We can even stick all the reused classes in here, too, to further reduce redundancy.

So this could be 

```ruby
def css_classes_for_comment(comment)
  return if comment.pseud.nil?
  return if comment.pseud.user.is_author_of?(comment.ultimate_parent) && comment.ultimate_parent.anonymous?
  
  "comment group user-#{comment.pseud.user.id}"
end
```

I know right now the single comment view does `comment group odd/even user-000`, but it's okay if it gets switched to `odd/even comment group user-000` -- the odd/even classes really should've been out front to begin with.

Reply:
I just realized this will clash a little bit with #4208, but I have to go afk -- I'll get back to you about the best way to handle that as soon as I can.